test/project_types/script/layers/application/build_script_test.rb

```diff
@@ -14,12 +14,13 @@
     let(:content) { 'content' }
     let(:extension_point_repository) { Script::Layers::Infrastructure::FakeExtensionPointRepository.new }
     let(:ep) { extension_point_repository.get_extension_point(extension_point_type) }
+    let(:task_runner) { Script::Layers::Infrastructure::AssemblyScriptTaskRunner.new(@context) }
```

Comment:
I think this `task_runner` should be a `Mock`, and it can be used to [expect](https://www.rubydoc.info/gems/minitest/Minitest/Mock#expect-instance_method) as well.

Reply:
You're right! I used a stub instead because it has accessor fields like `compiled_type` that we need to access and it was cleaner to use a stub there over a mock